crates/router/src/db/kafka_store.rs

```diff
@@ -3327,6 +3327,142 @@ impl GlobalStorageInterface for KafkaStore {
 }
 impl AccountsStorageInterface for KafkaStore {}
 
+#[async_trait::async_trait]
+impl hyperswitch_domain_models::merchant_connector_account::MerchantConnectorAccountInterface
+    for KafkaStore
+{
+    type Error = errors::StorageError;
+
+    #[cfg(feature = "v1")]
+    async fn find_merchant_connector_account_by_merchant_id_connector_label(
+        &self,
+        state: &KeyManagerState,
+        merchant_id: &id_type::MerchantId,
+        connector_label: &str,
+        key_store: &domain::MerchantKeyStore,
+    ) -> CustomResult<domain::MerchantConnectorAccount, Self::Error> {
+        hyperswitch_domain_models::merchant_connector_account::MerchantConnectorAccountInterface::find_merchant_connector_account_by_merchant_id_connector_label(&self.diesel_store, state, merchant_id, connector_label, key_store).await
```

Comment:
```suggestion
        merchant_connector_account::MerchantConnectorAccountInterface::find_merchant_connector_account_by_merchant_id_connector_label(&self.diesel_store, state, merchant_id, connector_label, key_store).await
```

do the same in other places too

Reply:
should we not be using methods from MerchantConnectorAccountInterface defined in hyperswitch_domain_models here?

Reply:
Import still the types and use it